File: TMessagesProj/src/main/java/tw/nekomimi/nekogram/utils/FileUtil.java
package tw.nekomimi.nekogram.utils;
import java.io.File;
public class FileUtil {
    public static File extLib(String name) { return new File(name); }
    public static void initDir(File dir) { if (dir != null && !dir.exists()) dir.mkdirs(); }
    public static void saveAsset(String asset, File target) {}
    public static void delete(File f) { if (f != null && f.exists()) f.delete(); }
    public static void deleteDirectory(File dir) {}
    public static long getDirSize(File dir) { return 0; }
}

File: TMessagesProj/src/main/java/tw/nekomimi/nekogram/utils/FileUtil.kt
package tw.nekomimi.nekogram.utils

import java.io.File

object FileUtil {

    @JvmStatic
    fun deleteDirectory(directoryToBeDeleted: File): Boolean = false

    @JvmStatic
    fun initDir(dir: File) {}

    @JvmStatic
    @JvmOverloads
    fun delete(file: File?, filter: (File) -> Boolean = { true }) {}

    @JvmStatic
    fun initFile(file: File) {}

    @JvmStatic
    fun readUtf8String(file: File) = ""

    @JvmStatic
    fun writeUtf8String(text: String, save: File) {}

    @JvmStatic
    fun saveAsset(path: String, saveTo: File) {}

    @JvmStatic
    val abi by lazy { "" }

    @JvmStatic
    fun extLib(name: String): File = File("")

    @JvmStatic
    fun saveNonAsset(path: String, saveTo: File) {}
}
